CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.6.0 (2020-05-08)
+
+- Added modern theme styles to the tab dropdown button. [#84](https://github.com/blackbaud/skyux-popovers/pull/84)
```

Comment:
Is "tab" correct? I see that in the code in the PR, but I don't see "tab" mentioned anywhere in the docs. We just refer to the dropdown button.

Reply:
Yes. There's a type of dropdown button specifically for the tab component.

Reply:
Sounds good. Just wanted to make sure.